appcues/src/main/java/com/appcues/ui/extensions/ModifierExt.kt

```diff
@@ -46,29 +45,26 @@ internal fun Modifier.outerPrimitiveStyle(
     gestureProperties: PrimitiveGestureProperties,
     isDark: Boolean,
     defaultBackgroundColor: Color? = null,
-    noSizeFillMax: Boolean = false,
 ) = this.then(
     with(component) {
         Modifier
             .margin(style.getMargins())
             .styleShadow(style, isDark)
-            .styleSize(style, noSizeFillMax)
+            .styleSize(style)
             .actions(id, gestureProperties)
             .styleCorner(style)
             .styleBorder(style, isDark)
             .styleBackground(style, isDark, defaultBackgroundColor)
-            .stylePrimitive(this)
     }
 )
 
-private fun Modifier.stylePrimitive(experiencePrimitive: ExperiencePrimitive) = this.then(
-    when (experiencePrimitive) {
-        is ImagePrimitive ->
-            Modifier
-                .animateContentSize()
-                .imageAspectRatio(experiencePrimitive.intrinsicSize)
-        else -> Modifier
-    }
```

Comment:
this was only used for the ImagePrimitive and only for the aspectRatio stuff, so moved to new function and changed the callsite.

Reply:
yea I did that locally as well. thanks!